std/algorithm/setops.d

```diff
@@ -3,6 +3,18 @@
 This is a submodule of $(MREF std, algorithm).
 It contains generic algorithms that implement set operations.
 
+The functions : $(LREF multiwayMerge), $(LREF multiwayUnion),
+$(LREF setDifference), $(LREF setIntersection), $(LREF setSymmetricDifference)
+expect a range of sorted ranges as input; providing a range of unsorted
+ranges will lead to a behaviour which is independent with the specification.
```

Comment:
Remove this subsentence - it's expected that it's UB if you violate the contract and this doesn't read well.